Common/src/config_structure.cpp

```diff
@@ -4458,7 +4458,7 @@ void CConfig::SetOutput(unsigned short val_software, unsigned short val_izone) {
                 cout << "The reference element size is: " << RefElemLength <<". "<< endl;
                 break;
               case VENKATAKRISHNAN_WANG:
-                cout << "Venkatakrishnan-Wang slope-limiting method, with constant: " << LimiterCoeff <<". "<< endl;
+                cout << "Venkatakrishnan-Wang slope-limiting method, with constant: 0.03."<< endl;
```

Comment:
Thanks for adding the coefficient back in.. can you please update these cout statements to print the value of LimiterCoeff again?

Reply:
Done!